app/diagnostics.go

```diff
@@ -148,10 +149,21 @@ func (a *App) trackActivity() {
 		activeUsersMonthlyCount = r.Data.(int64)
 	}
 
-	if ucr := <-a.Srv.Store.User().GetTotalUsersCount(); ucr.Err == nil {
+	if ucr := <-a.Srv.Store.User().Count(model.UserCountOptions{
+		IncludeBotAccounts: false,
+		IncludeDeleted:     true,
+	}); ucr.Err == nil {
 		userCount = ucr.Data.(int64)
 	}
 
+	if bc := <-a.Srv.Store.User().Count(model.UserCountOptions{
+		IncludeBotAccounts:  true,
+		IncludeDeleted:      false,
```

Comment:
nit: sometimes we leave the `nil` values implicit, and sometimes we make them explicit. Should we converge on one pattern or the other?